public/app/features/alerting/unified/rule-list/GrafanaRuleLoader.tsx

```diff
@@ -123,13 +125,14 @@ export function GrafanaRuleListItem({
 
   if (rulerRuleType.grafana.alertingRule(rulerRule)) {
     const promAlertingRule = rule && rule.type === PromRuleType.Alerting ? rule : undefined;
+    const instancesCount = sum(Object.values(promAlertingRule?.totals ?? {}));
```

Comment:
I _believe_ this is incorrect because the totals might contain `error` and `firing` which this will count as 2 distinct instances instead of 1. (error is not a state in and of itself).

I believe we have a util function somewhere to do correct counting on totals :)

Reply:
https://github.com/grafana/grafana/blob/43853e3410d5cfde5e918618231ab6d5d23a96b3/public/app/features/alerting/unified/components/rules/RuleStats.tsx#L83-L90